test/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { Link, VersionedLink } from '../lib/common-types'
+
+export function makeLink(type, id): Link<'Entry' | 'Asset'> {
```

Comment:
Given these functions can be used for more than just Entry and Asset should we instead have this be a generic?

```
export function makeLink<T extends string>(type: T, id): Link<T> {
```

Reply:
(agreed)